src/aura/RD2_EntryForm/RD2_EntryForm.cmp

```diff
@@ -14,14 +14,12 @@
     <aura:dependency resource="c:rd2EntryFormDonorSection" />
     <aura:dependency resource="c:rd2EntryFormCustomFieldsSection" />
 
-    <aura:attribute name="parentId" type="Id" description="Reference to the parent Id if open from related list" />
-
-    <aura:attribute name="modal" type="Object"
-        description="Reference to the modal in which RD2 entry form is displayed" />
-
-    <lightning:overlayLibrary aura:id="overlayLib" />
+    <aura:attribute name="parentId" type="Id" description="Reference to the parent Id if opened from related list" />
+    <aura:attribute name="renderModal" type="Boolean" default="false" description="Boolean to render Modal" />
 
     <aura:handler name="init" value="{!this}" action="{!c.doInit}" />
-
-    <c:utilDedicatedListener eventName="rd2EntryFormEvent" onreceiveevent="{!c.handleModalEvent}" />
+ 
+    <aura:if isTrue="{!v.renderModal}">
```

Comment:
When a New form is open and data populated, then click Cancel, Save or close the modal on the upper right corner, click New again, the form is pre-populated with data entered in the previous data form.

Moreover, when I saved *new* RD with Elevate credit card data populated, then viewed the RD tab again and clicked New the following screen was displayed:
<img width="876" alt="Screen Shot 2020-11-04 at 10 30 43 AM" src="https://user-images.githubusercontent.com/28743286/98154024-f3d2dc00-1e88-11eb-8cba-d494b05a447c.png">